src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs

```diff
@@ -411,6 +411,8 @@ private async Task<IActionResult> ExchangeAuthorizationCodeOrRefreshTokenGrantTy
             var principal = await _signInManager.CreateUserPrincipalAsync(user);
             var identity = (ClaimsIdentity) principal.Identity;
 
+            identity.AddClaim(new Claim("TrivestId", "Contoso"));
```

Comment:
?

Reply:
This is not supposed to be in there, going to remove it right away.